examples/ipfs-kad/README.md

```diff
@@ -1,31 +1,37 @@
 ## Description
 
 This example showcases the usage of **libp2p** to interact with the Kademlia protocol on the IPFS network.
-The code demonstrates how to perform Kademlia queries to find the closest peers to a specific peer ID.
-By running this example, users can gain a better understanding of how the Kademlia protocol operates and performs queries on the IPFS network.
+The code demonstrates how to: 
+ - perform Kademlia queries to find the closest peers to a specific peer ID
+ - insert PK records into the Kademlia DHT
+
+By running this example, users can gain a better understanding of how the Kademlia protocol operates and performs operations on the IPFS network.
 
 ## Usage
 
 The example code demonstrates how to perform Kademlia queries on the IPFS network using the Rust P2P Library.
+
+### Getting closes peers
+
 By specifying a peer ID as a parameter, the code will search for the closest peers to the given peer ID.
 
-### Parameters
+#### Parameters
 
 Run the example code:
 
 ```sh
-cargo run [PEER_ID]
+cargo run get-peers [PEER_ID]
```

Comment:
Did you try this command? I am surprised it doesn't need to be:

```suggestion
cargo run -- get-peers [PEER_ID]
```

Reply:
Either way works. Would you still like me to change it to the `--` version for clarity?